presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaRecordSet.java

```diff
@@ -47,9 +47,6 @@
 import static java.lang.String.format;
 import static java.util.Objects.requireNonNull;
 
-/**
- * Kafka specific record set. Returns a cursor for a topic which iterates over a Kafka partition segment.
```

Comment:
Is the second sentence really obvious?

(i'm fine with removal)

Reply:
It is not that much obvious but still it is obvious.